server/schema/dashboard.go

```diff
@@ -4,6 +4,7 @@ import (
 	"database/sql/driver"
 	"fmt"
 	"github.com/TheQuestionru/thequestion/server/types"
+	"github.com/abourget/teamcity"
```

Comment:
This library was used without good verification of its code. Later it turned out that it was crooked. I will replace it with another library, which I have already checked.